tests/test_api.py

```diff
@@ -385,3 +385,35 @@ def test_top_value_matches():
     assert "source" in df_match.columns
     assert "target" in df_match.columns
     assert "similarity" in df_match.columns
+
+def test_preview_domain():
```

Comment:
I would add a brief description of what each section is testing (e.g., describe that the function should not crash when the GDC has no values.)